projects/angular/src/layout/nav/_header.clarity.scss

```diff
@@ -588,95 +344,48 @@
       }
 
       input {
-        line-height: variables.$clr_baselineRem_1;
-        @include mixins.css-var(
-          margin,
-          clr-header-search-margin,
-          header-variables.$clr-header-nav-text-vertical-padding 0,
-          variables.$clr-use-custom-properties
-        );
+        line-height: tokens.$cds-global-space-9;
+        margin: auto;
       }
     }
 
     input[type='text'] {
       border: none;
       background: none;
-      @include mixins.css-var(
-        color,
-        clr-header-search-input-font-color,
-        header-variables.$clr-header-font-color,
-        variables.$clr-use-custom-properties
-      );
-
-      @include mixins.css-var(
-        height,
-        clr-header-search-height,
-        variables.$clr_baselineRem_1,
-        variables.$clr-use-custom-properties
-      );
+      color: header-variables.$clr-header-search-input-font-color;
+
+      height: header-variables.$clr-header-search-height;
 
       padding: 0;
-      @include mixins.css-var(padding-left, clr-header-search-input-padding, 0, variables.$clr-use-custom-properties);
+      padding-left: calc(header-variables.$clr-header-search-icon-size + 2 * header-variables.$clr-header-search-gap);
 
       vertical-align: middle;
 
       // Typography
-      @include mixins.css-var(
-        font-size,
-        clr-header-search-input-font-size,
-        inherit,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-weight,
-        clr-header-search-input-font-weight,
-        inherit,
-        variables.$clr-use-custom-properties
-      );
+      font-size: header-variables.$clr-header-search-input-font-size;
+      font-weight: header-variables.$clr-header-search-input-font-weight;
+
       &::placeholder {
-        @include mixins.css-var(
-          font-weight,
-          clr-header-search-placeholder-font-weight,
-          inherit,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          color,
-          clr-header-search-placeholder-text-color,
-          forms-variables.$clr-forms-placeholder-color,
-          variables.$clr-use-custom-properties
-        );
+        font-weight: header-variables.$clr-header-search-placeholder-font-weight;
+        color: header-variables.$clr-header-search-placeholder-text-color;
       }
 
       &:hover {
-        @include mixins.css-var(border, clr-header-search-border-hover, none, variables.$clr-use-custom-properties);
+        border: header-variables.$clr-header-search-border-hover;
       }
 
       &:focus-visible {
-        @include mixins.css-var(
-          outline,
-          clr-header-search-outline,
-          auto variables.$clr_baselineRem_1px Highlight,
-          variables.$clr-use-custom-properties
-        );
-
-        @include mixins.css-var(border, clr-header-search-border-active, none, variables.$clr-use-custom-properties);
+        outline: none; // do we need it?
```

Comment:
I believe is safe to delete it.
```suggestion
```

Reply:
We don't need the comment, but we do need the setting itself.
Otherwise we get an undesired glow effect.

Reply:
Keep it for now.